### PR TITLE
Automation and streamline code

### PR DESCRIPTION
**Pull request by yechenh**

### CODE REVIEW

File: streamline_training.py [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+    in the command line arguments."""
+    parser = argparse.ArgumentParser()
+    parser.add_argument("model_output_dir", help="the directory of the model output")
+    parser.add_argument("abs_path_to_lasertagger",
+                        help="absolute path to the folder where the lasertagger scripts are located")
+    parser.add_argument("abs_path_to_bert", help="absolute path to the folder where the pretrained BERT is located")
+    parser.add_argument("training_file", help="path to training samples")
+    parser.add_argument("tuning_file", help="path to tuning samples")
+
+    parser.add_argument("-vocab_size", type=int, help="vocab size", default=500)
```
[the remark this answers is not in the document]
Comment (yechenh) [reply streamline_training.py:221]: The ones starting with "-" are optional while the others are mandatory.

File: streamline_training.py [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+    with open(os.path.expanduser(args.training_file), 'r') as f:
+        num_training = sum(1 for _ in f)
+
+    if args.max_input_examples:
+        if args.max_input_examples <= num_training:
+            max_input_examples = str(args.max_input_examples)
+        else:
+            raise Exception("max input example > training data count")
+    else:
+        max_input_examples = str(int(num_training / 3))
```
[the remark this answers is not in the document]
Comment (yechenh) [reply streamline_training.py:95]: I was worried that using all training files might be overfitting. What would suggest? Thanks!

[the remark this answers is not in the document]
Comment (liangji-google) [reply streamline_training.py:95]: Normally, more training data is better. And you have flag to control number of training files. So the default value can be using all.
You can also add flag comment that default is using all.

[the remark this answers is not in the document]
Comment (yechenh) [reply streamline_training.py]: Done.

File: custom_predict.py [hunk 3]
```diff
@@ -0,0 +1,282 @@
...
+    Raise:
+        Exception: if the specified trained model does not exist at the GCP storage bucket 
+    """
+    for model in list_of_models:
+        if os.path.isdir(model):
+            print("-------model", model, "exists-------")
+        else:
+            print("-------downloading model", model, "-------")
+            try:
+                os.environ["model_name"] = "gs://trained_models_yechen/" + model
```
[the remark this answers is not in the document]
Comment (yechenh) [reply custom_predict.py]: Done.

**Pull request merged**